seastats/__init__.py

```diff
@@ -0,0 +1,7 @@
+from seastats.stats import GENERAL_METRICS
+from seastats.stats import GENERAL_METRICS_ALL
+from seastats.storms import STORM_METRICS
+from seastats.storms import STORM_METRICS_ALL
+
+SUGGESTED_METRICS = sorted(GENERAL_METRICS + STORM_METRICS)
+SUPPORTED_METRICS = sorted(GENERAL_METRICS_ALL + STORM_METRICS_ALL)
```

Comment:
there's still a grey zone here. These lists are not used yet. 
The real added value would be to have a common and higher level function that uses `get_stats` and `storm_metrics` depending on the parameters that are queried.